charts/seed-bootstrap-crds/charts/hvpa/values.yaml

```diff
@@ -0,0 +1,3 @@
+enabled: false
```

Comment:
Currently there is no check on the HVPA CRD template and seed bootstrap will always create the HVPA CRD (no matter the value of the feature gate).
We have to add if statement to the CRD itself. Or alternative approach is to reuse the mechanism of `requirements.yaml` file that lists the dependencies with appropriate conditions for enablement. I think most of our charts with dependencies follow this approach. I was just wondering whether it would be possible to reuse this `requirements.yaml` approach also for the newly introduced `seed-bootstrap-crds` chart. I think the benefit of this approach is that we don't have to maintain if statement for enablement for each and every template file and it is kind of controlled by the condition in the requirements file. WDYT? (I see that the vpa chart is referenced by another chart, so if the approach does not work, then I am fine with simply adding if statement for the HVPA CRD template to tackle this issue).